record_tx_strategy_internal_incoming_tx.go

```diff
@@ -14,9 +14,9 @@ type internalIncomingTx struct {
 	allowBroadcastErrors bool // only BEEF cannot allow for broadcast errors
 }
 
-func (strategy *internalIncomingTx) Execute(ctx context.Context, c ClientInterface, opts []ModelOps) (*Transaction, error) {
+func (strategy *internalIncomingTx) Execute(ctx context.Context, c ClientInterface, _ []ModelOps) (*Transaction, error) {
 	logger := c.Logger()
-	logger.Info(ctx, fmt.Sprintf("InternalIncomingTx.Execute(): start, TxID: %s", strategy.Tx.ID))
+	logger.Info().Msgf("InternalIncomingTx.Execute(): start, TxID: %s", strategy.Tx.ID)
```

Comment:
`InternalIncomingTx.Execute()` Can be added to `Str()`

Reply:
Removed function name from logs